### PR TITLE
Eliminate all usages of Guava from the project

### PR DESCRIPTION
**Pull request by iravid**
See b5c901be for the actual changes

### CODE REVIEW

File: server/src/main/scala/org/ensime/server/Server.scala [hunk 1]
```diff
@@ -129,7 +129,7 @@ object Server extends AkkaBackCompat {
       throw new RuntimeException(s".ensime file ($ensimeFile) not found")
 
     implicit val config: EnsimeConfig = try {
-      EnsimeConfigProtocol.parse(Files.toString(ensimeFile, Charsets.UTF_8))
+      EnsimeConfigProtocol.parse(new String(Files.readAllBytes(ensimeFile.toPath), StandardCharsets.UTF_8))
```
Comment (fommil) [inline server/src/main/scala/org/ensime/server/Server.scala]: I'm sure there is a better way to do this by adding the functionality onto EnsimeRawFile

File: util/src/main/scala/org/ensime/util/file.scala [hunk 2]
```diff
@@ -64,20 +63,20 @@ package object file {
     def outputStream(): OutputStream = new FileOutputStream(file)
 
     def createWithParents(): Boolean = {
-      GFiles.createParentDirs(file)
+      file.getCanonicalFile.getParentFile.mkdirs()
```
Comment (fommil) [inline util/src/main/scala/org/ensime/util/file.scala]: prefer `.canon` to `getCanonicalFile`

File: util/src/test/scala/org/ensime/util/FileSpec.scala [hunk 3]
```diff
@@ -56,7 +56,7 @@ class FileSpec extends FlatSpec with Matchers {
     out.write("hello".getBytes())
     out.close()
 
-    Files.toString(file, Charsets.UTF_8) shouldBe "hello"
+    new String(Files.readAllBytes(file.toPath), StandardCharsets.UTF_8) shouldBe "hello"
```
Comment (fommil) [inline util/src/test/scala/org/ensime/util/FileSpec.scala]: you can use our helper here

**Comment by fommil** [comment]
@iravid sorry looks like you have some conflicts

**Comment by iravid** [comment]
Yeah, couldn't find the time to get this over the line the last few weeks.
Will try this weekend or the next :-)

**Comment by fommil** [comment]
#1740

**Pull request closed without merging**